app/utils/sendEndringsmelding.ts

```diff
@@ -0,0 +1,35 @@
+import { Session, json } from '@remix-run/node';
+import postResponseMock from '~/mock/postResponseMock';
+import { postMedToken } from '~/server/authorization';
+import { EMiljø } from '~/typer/miljø';
+
+const STI: string = '/api/send-inn/ba';
+const LOKAL_URL_BACKEND: string = 'http://localhost:8099' + STI;
+//const API_URL_BACKEND: string ='https://nav-familie-endringsmelding-api.fly.dev/' + STI;
+
+export async function sendEndringsmelding(
+  endringsmelding: string,
+  session: Session,
+): Promise<Response> {
+  const requestInfo = new Request(LOKAL_URL_BACKEND, {
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+    },
+    method: 'POST',
+  });
+
+  switch (process.env.ENV) {
+    case EMiljø.LOKAL:
+      return await postMedToken(
+        session,
+        LOKAL_URL_BACKEND,
+        requestInfo,
+        JSON.stringify(endringsmelding),
+      );
+    //case EMiljø.PRODUKSJON:
+    //Her kommer case for bruke av API i produksjon
+    default:
+      return json(postResponseMock);
```

Comment:
Usikker på hvordan `json` fungerer her, men denne må returnere Response, så tror dette vil funke for å få samme type objekt som ved et api-kall;

`Response.json(postResponseMock)`